doc/irmin-pack/design/layered_store.md

```diff
@@ -301,13 +301,13 @@ disk or not. If it was, then the main process restarts, reads
 `control` (which was successfully renamed from `control.nnnn` ), and

```

Comment:
Should li. 299 read: `control.nnnn` over the current `control` was recorded persistently on
disk or it wasn't. 

Would you please clarify this sentence? It doesn't make sense as it's written.

Reply:
(marked as resolved because it's easier to discuss in your previous comment that contains the rest of the sentence)